### DISCUSSION TITLE
Notifications URL

### DISCUSSION THREAD

**Discussion by excaliburzarau** [containrrr/shoutrrr#367] [Q&A] [answered]
Hello guys,

I have been trying to setup the Notification URL but i am not able to see how can i configure an hotmail email to send the notifications. I have it already setup on a different application and it is working fine. I believe it might be possible that the authentication method is incorrect but i already tried them all.

This is the Env Variable i am using:

NOTIFICATION_URLS=smtp://senderemail%40hotmail.com:password123@smtp.office365.com:587/?fromAddress=senderemail%40hotmail.com&toAddresses=destinyemail%40hotmail.com&auth=Unknown&starttls=Yes

I do believe this should be enough by reading the Doc but i am unable to make it successfully send the email.
I did use the generator to generate the URL.
I have tried to figure out the configuration for the Outlook IMAP regarding the Authentication, ClientHost and Encryption but without success. I set the Authentication to Unknown, the ClientHost to Auto and the Encryption also to Auto but it does not seem to be working for some reason.

Can you guys give me some pointers?
The error i receive is the one bellow.

Backup procedure ran successfully, but an error ocurred calling the registered hooks: runHooks: error running hook: notify: error notifying: sendNotification: error sending message: failed to send using smtp: timed out"

Thank you all in advance.

**Comment by piksel** [comment, marked as answer]
Unfortunately, the authentication method that Microsoft supports isn't supported by the go E-mail client. There is a feature request issue for it.

**Reply by excaliburzarau** [reply]
Thank you for your reply.

After some troubleshooting i was able to create an account with GMX and generate an URL for it.
It should be enough for the purpose.

I here leave a generated URL that is working at the moment for GMX.

smtp://sender%40gmx.com:password123@mail.gmx.com:587/?fromaddress=sender%40gmx.com&fromname=EmailSender&subject=Data+Backup+Test&toaddresses=destiny%40hotmail.com

Best Regards and thank you for your reply.
